### PR TITLE
Impact Analysis - CMS69

### PR DESCRIPTION
**Pull request by soy-park**
Impact analysis of QICore6 uplift on CMS69.

- Added comments for each define statements that need to be updated:
   - Observation resource to SimpleObservation
   - ObservationNotDone to ObservationCancelled
   - Condition resource to ConditionProblemsHealthConcerns

### CODE REVIEW

File: input/cql/PCSBMIScreenAndFollowUpFHIR.cql [hunk 1]
```diff
@@ -131,6 +150,9 @@ define "Low BMI Interventions Ordered":
         )
   )
 
+/*
+@QICore6Uplift: Update 'Condition to 'ConditionProblemsHealthConcerns'
```
Comment (guptava) [inline input/cql/PCSBMIScreenAndFollowUpFHIR.cql:154]: Looks like a Procedure, so not sure about using Condition. No update required in my opinion.

**Review by guptava — COMMENTED** [review]
Added some comments to review.

File: input/cql/PCSBMIScreenAndFollowUpFHIR.cql [hunk 2]
```diff
@@ -86,6 +93,9 @@ define "High BMI and Follow up Provided":
       union "High BMI Interventions Performed" ) HighBMIInterventions
       such that Coalesce(HighBMIInterventions.performed.toInterval(), HighBMIInterventions.authoredOn.toInterval()) starts during day of "Measurement Period"
 
+/*
+@QICore6Uplift: Update 'Condition to 'ConditionProblemsHealthConcerns'
```
[the remark this answers is not in the document]
Comment (soy-park) [reply input/cql/PCSBMIScreenAndFollowUpFHIR.cql:97]: Further down in the code, there is a Condition resource type that is being used

File: input/cql/PCSBMIScreenAndFollowUpFHIR.cql [hunk 3]
```diff
@@ -55,6 +59,9 @@ define "Denominator Exceptions":
   exists "Medical Reason for Not Documenting a Follow up Plan for Low or High BMI"
     or exists "Medical Reason or Patient Reason for Not Performing BMI Exam"
 
+/*
+@QICore6Uplift: Update 'observation-bmi' to 'SimpleObservation'
```
[the remark this answers is not in the document]
Comment (soy-park) [reply input/cql/PCSBMIScreenAndFollowUpFHIR.cql:63]: I didn't find a specific resource type called 'observation-bmi' which is probably why there's a red underline with the VS Code plugin. Changing this to 'SimpleObservation' removes the error. If you find a more specific Observation that pertains to BMI, let me know!

File: input/cql/PCSBMIScreenAndFollowUpFHIR.cql [hunk 4]
```diff
@@ -118,6 +134,9 @@ define "Medical Reason or Patient Reason for Not Performing BMI Exam":
           or NoBMI.notDoneReason in "Medical Reason"
       )
 
+/*
+@QICore6Uplift: Update 'Condition to 'ConditionProblemsHealthConcerns'
```
[the remark this answers is not in the document]
Comment (soy-park) [reply input/cql/PCSBMIScreenAndFollowUpFHIR.cql:138]: Further down in the code, there is a Condition resource type that is being used.

File: input/cql/PCSBMIScreenAndFollowUpFHIR.cql (hunk 1 above)
Comment (soy-park) [reply input/cql/PCSBMIScreenAndFollowUpFHIR.cql:154]: Further down in the code, there is a Condition resource type that is being used.

**Review by guptava — APPROVED** [review]

**Pull request merged**